x-pack/plugins/lens/public/xy_visualization/expression.tsx

```diff
@@ -179,6 +179,12 @@ export const xyChart: ExpressionFunctionDefinition<
       help: 'Layers of visual series',
       multi: true,
     },
+    curveType: {
+      types: ['number'],
+      help: i18n.translate('xpack.lens.xyChart.curveType.help', {
+        defaultMessage: 'Define how curve type is rendered for a line chart',
```

Comment:
Can we pass the enum here instead of the number (and resolve it in the component using `CurveType[arg]`)? That seems easier to follow (and I'm somehow scared of the numbers changing and nobody notices)

Reply:
i am not sure how enum will work here. can you give an example?

Reply:
done !!